misc/python/materialize/cli/mzcompose.py

```diff
@@ -611,7 +611,23 @@ def handle_composition(
 BuildCommand = DockerComposeCommand("build", "build or rebuild services")
 ConfigCommand = DockerComposeCommand("config", "validate and view the Compose file")
 CreateCommand = DockerComposeCommand("create", "create services", runs_containers=True)
-DownCommand = DockerComposeCommand("down", "stop and remove resources")
+
+
+class DownCommand(DockerComposeCommand):
+    def __init__(self) -> None:
+        super().__init__(
+            "down",
+            "clean up containers from the composition",
```

Comment:
The descriptions were copied directly from Docker Compose. If we're going to change it, let's change it to the new text from Docker Compose v2, which is more clear?

"Stop and remove containers, networks"